static/js/theme-map/VerticalFullPageMapOrchestrator.js

```diff
@@ -159,9 +167,99 @@ class VerticalFullPageMapOrchestrator extends ANSWERS.Component {
       this.searchThisArea();
     });
 
+    this.setupCssForBreakpoints();
     this.addMapComponent();
   }
 
+  /**
+   * Properly set CSS classes for mobile and desktop
+   */
+  setupCssForBreakpoints () {
+    if (!this.isMobile()) {
+      this.updateCssForDesktop();
+    }
+
+    const mediaQuery = window.matchMedia(`(max-width: ${this.mobileBreakpointMax}px)`);
+    mediaQuery.addEventListener('change', () => {
```

Comment:
Does this listener persist outside of this function, given the variable is defined within this scope? Do we want to move this to a class variable so it can be used in both here and `isMobile`?

Reply:
updated to put the media query in the class